lib/fluent/plugin/out_redis.rb

```diff
@@ -29,8 +30,13 @@ def configure(conf)
     def start
       super
 
-      @redis = Redis.new(:host => @host, :port => @port,
-                         :thread_safe => true, :db => @db_number)
+      options = {:host => @host, :port => @port,
+                 :thread_safe => true, :db => @db_number}
```

Comment:
Good point.
But `{:key => value}` Hash style is used in this plugin.
If we want to use `{key: value}`, we should use without rocket hash operator style entirely in this plugin.
So, I would like to keep it as-is for now.